src/components/ForexRatesTab.js

```diff
@@ -9,25 +9,23 @@ import SnackbarContentWrapper from './SnackbarUtils';
 import { getForexRates } from '../api';
 
 export const stringRateFromDecimalRateAndInteger = (decimalRate, integer) => {
+  const forexRate = String(integer);
```

Comment:
Any reason we dont use the https://github.com/mojaloop/ml-number library for this?

I believe it already has the functionality to convert integers to string, and also set the "fixed" decimals?